lib/utils/jquery.js

```diff
@@ -0,0 +1,33 @@
+const { get } = require('../utils/get');
+
+const JQUERY_ALIASES = ['$', 'jQuery'];
+
+function isJQueryCallee(node, name = null) {
+  const calleePropertyNameisJquery = get(node, 'object.callee.property.name') === '$';
```

Comment:
I think these should use `JQUERY_ALIASES` (instead of hard coding to `$`).

Something like:

```js
const calleePropertyNameisJquery = JQUERY_ALIASES.includes(get(node, 'object.callee.property.name'));
```

Reply:
Done